runtime/commonMain/src/kotlinx/benchmark/Utils.kt

```diff
@@ -0,0 +1,7 @@
+package kotlinx.benchmark
+
+internal expect fun String.readFile(): String
+
+internal expect fun String.writeFile(text: String)
+
+internal expect inline fun measureTime(block: () -> Unit): Long
```

Comment:
Please document units in which the returned Long is expressed.

Reply:
may be it's better to just rename it to `measureNanoseconds`
BTW: is it ok to use Long here? I mean, it's benchmarking code, and `Long` in Kotlin/JS is an object, not a primitive.

Reply:
Ok, I will rename that.
Yes, this Long is used out of timer scope.